internal/provider/resource_gitlab_group_project_file_template_test.go

```diff
@@ -0,0 +1,139 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/xanzy/go-gitlab"
+	"testing"
+)
+
+func TestAccGitlabGroupProjectFileTemplate_basic(t *testing.T) {
+	rInt := acctest.RandInt()
+	var parentGroup gitlab.Group
+	var templateProject gitlab.Project
```

Comment:
Let's create the test group and project outside of TF and focus in the TF config on "what is under test" ...